bc_obps/registration/tests/test_endpoints.py

```diff
@@ -90,10 +90,10 @@ def test_post_new_operation(self, client):
         )
         post_response = client.post(self.endpoint, content_type=content_type_json, data=mock_operation.json())
         assert post_response.status_code == 200
-        assert post_response.json() == {"name": "Springfield Nuclear Power Plant"}
+        assert post_response.json() == {"name": "Springfield Nuclear Power Plant", "id": 2}
```

Comment:
Not a Django testing expert but that is the id that is being created and returned in this test. I assume a test above it is creating `"id: 1"`

Reply:
I don't think it's safe to assume the correct value is what the test returns. If you just run that single test do you get id=1?

Reply:
It's not 100% guaranteed that this `id` value will always be 2, because the order in which unit tests are run can vary. In this test case we don't care what the specific valud of `id` is, we just want to make sure that the id key-value pair is returning in the json, so I'd recommend replacing this with something like 

```
assert post_response.json().get('name') == 'Springfield...' 
assert post_response.json().get('id') is not None
```